Xamarin.PropertyEditing/ViewModels/PropertyViewModel.cs

```diff
@@ -439,12 +439,13 @@ private async void UpdateAutocomplete (string value)
 					tasks.Remove (results);
 
 					if (list == null) {
-						list = results.Result;
+						list = await results;
 						common = new HashSet<string> (list);
 					} else
-						common.IntersectWith (results.Result);
+						common.IntersectWith (await results);
 				} while (tasks.Count > 0 && !cancel.IsCancellationRequested);
 
+				cancel.ThrowIfCancellationRequested ();
```

Comment:
This needs a test prove the new behavior, can you add one for @alanmcgovern, @CartBlanche ?

Reply:
Sure, I'll give it a try.